libs/vkd3d/vkd3d_private.h

```diff
@@ -2099,12 +2099,20 @@ struct d3d12_command_list
     bool is_valid;
     bool debug_capture;
     bool has_replaced_shaders;
-    bool has_valid_index_buffer;
+
+    struct
+    {
+        VkBuffer buffer;
+        VkDeviceSize offset;
+        DXGI_FORMAT dxgi_format;
+        VkIndexType vk_type;
+        bool is_non_null;
```

Comment:
couldn't we use the VkBuffer being `VK_NULL_HANLDE` for this case instead? This seems somewhat redundant now.

Reply:
Fixed.